pkg/network/bridge/plugin.go

```diff
@@ -114,22 +127,43 @@ func (nbdp *NetworkBridgeDevicePlugin) Allocate(ctx context.Context, r *pluginap
 
 	for _, req := range r.ContainerRequests {
 		var devices []*pluginapi.DeviceSpec
+		var vnics []vnic
 		for _, nic := range req.DevicesIDs {
 			dev := new(pluginapi.DeviceSpec)
 			assignmentPath := getAssignmentPath(nbdp.bridge, nic)
 			dev.HostPath = fakeDevicePath
 			dev.ContainerPath = assignmentPath
 			dev.Permissions = "r"
 			devices = append(devices, dev)
+			vnics = append(vnics, vnic{nic, protocolEthernet})
 
 			nbdp.assignmentCh <- &Assignment{
 				nic,
 				assignmentPath,
 				time.Now(),
 			}
+
+		}
+
+		vnicsPerInterface := networkInterfaceRessources{
+			Name:       fmt.Sprintf("%s/%s", ResourceNamespace, nbdp.bridge),
+			Interfaces: vnics,
 		}
+
+		marshalled, err := json.Marshal(vnicsPerInterface)
```

Comment:
@yuvalif  should we put vnicsPerInterface in a list, to enable multiple interfaces like
[{"name": "network-bridge.dp.io/red", "interfaces": [{"name": "eth-red01", "protocol": "Ethernet"},{"name": "eth-red02", "protocol": "Ethernet"}]}, {"name": "network-bridge.dp.io/blue", "interfaces": [{"name": "eth-blue01", "protocol": "Ethernet"}]}]

Reply:
Missed this hidden comment. I don't think there is a way using pure DP. Not only multiple plugins, but it is also not documented whether allocation on one plugin is done using single Allocate call. It would be safest if we used a random suffix for the variable.